src/scalaDebugger.ts

```diff
@@ -247,3 +247,14 @@ export interface DebugSession {
   name: string;
   uri: string;
 }
+
+export interface DebugDiscoveryParams {
```

Comment:
@ckipp01 should we move these classes to the language client?

Reply:
Yea probably. I'll open a pr for that. Also, should we put in the other optional fields?

```
 "args"?: string[],
 "jvmOptions"?: string[],
 "env"?: {}, // <-- I actually don't know how this is is done in TS to signify it's an object with unknown k v pairs.
 "envFile"?: string
```

Reply:
Sure, we can. I haven't added them since they would require using `launch.json`, which I think might not work when running current file.  Also it's kind of weird to set the same options for each file we would run. I think if a user requires arguments etc. then it makes sense to create a dedicated run configuration.

Reply:
Yea true sounds good.

Reply:
https://github.com/scalameta/metals-languageclient/pull/248